### ISSUE TITLE
Upstream PRs not found by rr/rrr

### ISSUE THREAD

**Issue by wjmelements** [wjmelements/scripts#13]
```
$ rrr
usage: rrr [-f] [num...]
No existing pull requests for wjmelements/solidity:bytes-complex-type
```

**Comment by wjmelements** [comment]
I believe this is because I only check the pull requests at `origin`.

**Comment by wjmelements** [comment]
fixed in 01312aac5bb39659a846c58318b4f4e49ec27cd2
